include/simdjson/inline/document_stream.h

```diff
@@ -5,10 +5,70 @@
 #include <algorithm>
 #include <limits>
 #include <stdexcept>
-
 namespace simdjson {
 namespace dom {
 
+#ifdef SIMDJSON_THREADS_ENABLED
+namespace {
+  // Putting those inside the stage1_worker class may create problems with move constructors
+  // since a mutex is not supposed to be moved.
+  std::mutex m{};
+  std::condition_variable cv{};
```

Comment:
Any chance of a name on these that describes roughly what people will use them to wait on?

Reply:
I can fix that but now that you made me realize that I can't put them there... I have to do a redesign.

Reply:
I changed the name but they are meant to be private variables. You are only ever supposed to touch those if you want to hack the worker class.

Reply:
Yeah. It's never ever going to be public API.